lib/spack/spack/directives.py

```diff
@@ -289,9 +289,10 @@ def extends(spec, **kwargs):
 
     """
     def _execute(pkg):
-        if pkg.extendees:
-            msg = 'Packages can extend at most one other package.'
-            raise DirectiveError(msg)
+        # if pkg.extendees:
+        #     directive = 'extends'
+        #     msg = 'Packages can extend at most one other package.'
+        #     raise DirectiveError(directive, msg)
```

Comment:
Is this safe to remove? It causes problems for packages that require:
```python
extends('python', ignore='bin/.*')
```
since there is already:
```python
extends('python')
```
in the base class.